apis/python/src/tiledbsc/scgroup.py

```diff
@@ -355,6 +361,8 @@ def write_obs_or_var(self, obs_or_var_data, obs_or_var_name: str):
             offsets_filters=offsets_filters,
             attr_filters=attr_filters,
             dim_filters=dim_filters,
+            capacity=100000,
+            tile=extent,
```

Comment:
Tile extent does not have the same effect for sparse as it does dense. For sparse this _only_ effects sorting. Capacity is the main things to be concerned with here.